.github/workflows/create-tag.yml

```diff
@@ -64,8 +72,44 @@ jobs:
 
       - name: Create and push tag
         env:
-          TAG: ${{ github.event.inputs.tag }}
+          TAG: 'v${{ github.event.inputs.tag }}'
           BRANCH: ${{ github.event.inputs.branch }}
         run: |
-          git tag "v$TAG"
-          git push origin "v$TAG"
+          if [ -z "$(git tag -l $TAG)" ]; then
+            git tag "$TAG"
+            git push origin "$TAG"
```

Comment:
Do we need to do this? According to the [docs for `gh release create`](https://cli.github.com/manual/gh_release_create):

> If a matching git tag does not yet exist, one will automatically get created from the latest state of the default branch. Use --target to point to a different branch or commit for the automatic tag creation

Reply:
I think so. I'm adding the option to create the draft release notes as optional step, so I still need this if I don't want to create the draft release notes.